tiagosatur/bluejack
# FEATURE | Create view pile page

**What this PR does?**
It adds the view pile page. Where the user cards are fetched, reorganized from higher to lower according to the rotation card, and show full house combinations if they exist

**How to test?**
- Register cards and submit
- See if they are organized according to the playbook from higher to lower, suits have higher weight than ranks.
- If was inputted cards with three ranks and two ranks equal, should appear a full house combo in the view.
- You can copy and paste the URL address with the deck_id and the page should work as well
